tests/Integration/Behaviour/Features/Scenario/Order/order_cart_rules.feature

```diff
@@ -770,3 +769,55 @@ Feature: Order from Back Office (BO)
+    Given there is a product in the catalog named "product_expensive" with a price of 123.00 and 100 items in stock
+    And there is a product in the catalog named "product_cheap" with a price of 10.00 and 100 items in stock
+    When I create an empty cart "dummy_cart_cart_rule_cheapest" for customer "testCustomer"
+    And I add 1 products "product_expensive" to the cart "dummy_cart_cart_rule_cheapest"
+    And I select "US" address as delivery and invoice address for customer "testCustomer" in cart "dummy_cart_cart_rule_cheapest"
+    Then cart "dummy_cart_cart_rule_cheapest" should contain 1 products
+    ## Create an order with the expensive product
+    When I add order "bo_order1" with the following details:
+      | cart                | dummy_cart_cart_rule_cheapest |
+      | message             |                               |
+      | payment module name | dummy_payment                 |
+      | status              | Payment accepted              |
+    Then order "bo_order1" should have 1 products in total
+    And order "bo_order1" should have 0 cart rule
+    And order "bo_order1" should have following details:
+      | total_products           | 123.000 |
+      | total_products_wt        | 130.380 |
+      | total_discounts_tax_excl | 0.000   |
+      | total_discounts_tax_incl | 0.000   |
+      | total_paid_tax_excl      | 130.000 |
+      | total_paid_tax_incl      | 137.80  |
+      | total_paid               | 137.80  |
+      | total_paid_real          | 137.80  |
+      | total_shipping_tax_excl  | 7.000   |
+      | total_shipping_tax_incl  | 7.42    |
+    And order "bo_order1" should have invoice
+    ## Create a new cart rule
+    When I want to create a new cart rule
+    And I specify its name in default language as "Cart Rule 50% which excludes discounted products and applies to cheapest product"
+    And I specify its "description" as "None"
+    And I specify its "code" as " "
+    And I specify that its active from "2019-01-01 11:00:00"
+    And I specify that its active until "2040-01-01 12:00:00"
+    And I specify that its "quantity" is "100"
+    And I specify that its "quantity per user" is "100"
+    And I specify that its "priority" is "1"
+    And its minimum purchase amount in currency "USD" is "0.0"
+    And its minimum purchase amount is tax included
+    And its minimum purchase amount is shipping included
+    And I specify its status as enabled
+    And it gives a percentage reduction of "50" which excludes discounted products and applies to cheapest product
+    When I save it
+    ## Add the product to the order
+    When I add products to order "bo_order1" without invoice and the following products details:
+      | name          | product_cheap     |
+      | amount        | 1                 |
+      | price         | 10                |
+    Then order "bo_order1" should have 2 products in total
+    And order "bo_order1" should have 1 cart rule
```

Comment:
Maybe you should add a bit more details to check the cart rule values? Like cart rule amount and order details with discount values

Reply:
Fixed